backend/src/main/java/com/java/academy/gym/model/BaseEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public class BaseEntity {
```

Comment:
Good idea to create a base entity but better to make it abstract and implement Serializable interface that it helps to use entity as a DTO object